components/automate-ui/src/app/components/page-picker/page-picker.component.html

```diff
@@ -1,35 +1,83 @@
 <ng-container *ngIf="selectablePages.length > 1">
-  <chef-button secondary
-    class="page-picker-item page-picker-first"
-    (click)="onItemTap(first)"
-    [disabled]="isNull(prev)">
-    <chef-icon>first_page</chef-icon>
-  </chef-button>
-  <chef-button secondary
-    class="page-picker-item page-picker-prev"
-    (click)="onItemTap(prev)"
-    [disabled]="isNull(prev)">
-    <chef-icon>chevron_left</chef-icon>
-  </chef-button>
-  <chef-button
-    *ngFor="let item of selectablePages; trackBy: trackBy"
-    class="page-picker-item"
-    [attr.primary]="item === page"
-    [attr.secondary]="item !== page"
-    [attr.disabled]="selectablePages.length <= 1"
-    (click)="onItemTap(item)">
-    {{ item }}
-  </chef-button>
-  <chef-button secondary
-    class="page-picker-item page-picker-next"
-    (click)="onItemTap(next)"
-    [disabled]="isNull(next)">
-    <chef-icon>chevron_right</chef-icon>
-  </chef-button>
-  <chef-button secondary
-    class="page-picker-item page-picker-last"
-    (click)="onItemTap(last)"
-    [disabled]="isNull(next)">
-    <chef-icon>last_page</chef-icon>
-  </chef-button>
+  <ng-container *ngIf="!forDesktop">
+    <chef-button secondary
+      class="page-picker-item page-picker-first"
+      (click)="onItemTap(first)"
+      [disabled]="isNull(prev)">
+      <chef-icon>first_page</chef-icon>
+    </chef-button>
+    <chef-button secondary
+      class="page-picker-item page-picker-prev"
+      (click)="onItemTap(prev)"
+      [disabled]="isNull(prev)">
+      <chef-icon>chevron_left</chef-icon>
+    </chef-button>
+    <chef-button
+      *ngFor="let item of selectablePages; trackBy: trackBy"
+      class="page-picker-item"
+      [attr.primary]="item === page"
+      [attr.secondary]="item !== page"
+      [attr.disabled]="selectablePages.length <= 1"
+      (click)="onItemTap(item)">
+      {{ item }}
+    </chef-button>
+    <chef-button secondary
+      class="page-picker-item page-picker-next"
+      (click)="onItemTap(next)"
+      [disabled]="isNull(next)">
+      <chef-icon>chevron_right</chef-icon>
+    </chef-button>
+    <chef-button secondary
+      class="page-picker-item page-picker-last"
+      (click)="onItemTap(last)"
+      [disabled]="isNull(next)">
+      <chef-icon>last_page</chef-icon>
+    </chef-button>
+  </ng-container>
+
+  <!-- new styles being used only for desktop at the moment -->
+  <ng-container *ngIf="forDesktop">
+    <div class="bx--row">
+      <div class="column information-container ">
+        <div class="per-page-container">
+          <span class="items-per">Items per page: </span>
+            <mat-form-field appearance="none">
+              <mat-select panelClass="chef-dropdown" [(ngModel)]="perPage">
```

Comment:
Try to avoid using two-way binding -- `[(ngModel)]`. One goal of Automate's design is to use one-way binding throughout. 
For reference: 

- The "One-way Binding" section of [this article](http://www.techtutorhub.com/article/Simple-Example-of-One-Way-and-Two-Way-Binding-in-Angular-8/60) shows how to split it for a text field.
- The plunker attached to [this comment](https://github.com/ngrx/store/issues/228#issuecomment-249553535) demonstrates how to achieve one-way data flow for a checkbox. 

Interestingly, the many other comments on that last link talk about how clunky it is to do one-way data flow at times.

Reply:
Thanks for catching this.  Wild enough - I had actually implemented one way data flow, but left in the ability to two way it 🤷‍♂️ .  Appreciate the added info too!